f3read.c

```diff
@@ -164,6 +164,8 @@ static void validate_file(const char *path, int number,
 
 	/* Obtain initial time. */
 	assert(!gettimeofday(&t1, NULL));
+	pt1 = t1;
+	pt1.tv_sec -= 1000;
```

Comment:
Hi @sgerwk,

This patch is not fixing a bug. In fact, it is not changing behavior either.

`pt1` is initialized with -1s, and its first use is at line `if (delay_ms(&pt1, &pt2) >= 200) {`. Due to the initialization of `pt1`, `delay_ms(&pt1, &pt2)` must return more than 1000, which is greater than 200. So the first test is always true. Your patch accomplishes the same thing with two more lines of code.